app/assets.js

```diff
@@ -1,101 +1,102 @@
-var config = require('./config');
-var log = require('bole')('assets');
-var path = require('path');
+const Bluebird = require('bluebird');
+const fs = Bluebird.promisifyAll(require('fs'));
+const glob = Bluebird.promisify(require('glob').glob);
```

Comment:
I see that `Bluebird.promisifyAll()` adds the `Async` suffix. I wonder if for consistency and for the sake of being explicit we should add this `Async` suffix to the name of all the "promisified" - just a though.

Reply:
No - that would lead to ever longer function names. I can't help what Bluebird does to wrapped functions, but I don't agree with it.